### PR TITLE
Reshuffle docs/ versions/ and ush/ contents - release/satingest.v4.2.1_RM

### PR DESCRIPTION
**Pull request by ilianagenkova**
-move build.sh from top level dir to ush/
-move VERSION to versions/
-renamed release_notes/ to docs/ 
-moved README.md to docs/
-updated Release_Notes.txt, CMakeLists.txt, build.sh and README.md

### CODE REVIEW

**Review by SteveStegall-NOAA — COMMENTED** [review]
Looks good to me.  Thanks.

File: ush/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 list(APPEND ush_files
+  build.sh
```
Comment (aerorahul) [inline ush/CMakeLists.txt]: This should not be installed.

File: docs/Release_Notes.txt [hunk 2]
```diff
@@ -1,229 +1,38 @@
...
-   fix/tovs.atovs
-   fix/tovs.hirs4
-   fix/tovs.mhs
-   jobs/CMakeLists.txt
-   jobs/JISATINGEST
-   modulefiles/satingest_common.lua
-   modulefiles/satingest_hera.lua
-   modulefiles/satingest_orion.lua
-   modulefiles/satingest_wcoss2.lua
  A release_notes/CMakeLists.txt
```
Comment (aerorahul) [inline docs/Release_Notes.txt:20]: Please remove this `CMakeLists.txt`.

File: ush/build.sh [hunk 3]
```diff
@@ -3,7 +3,8 @@
 set -eux
 
 # Location of PWD and package source directory.
-pkg_root=`dirname $(readlink -f $0)`
+#pkg_root=`dirname $(readlink -f $0)`
```
Comment (aerorahul) [inline ush/build.sh]: remove commented lines.

**Review by aerorahul — CHANGES_REQUESTED** [review]
A couple of requested changes.

File: docs/Release_Notes.txt (hunk 2 above)
Comment (ilianagenkova) [reply docs/Release_Notes.txt:20]: This line moved as a line number.

**Comment by ilianagenkova** [comment]
@SteveStegall-NOAA and @aerorahul , I can't close the PR unless one of you approves the PR. could you do that? thanks

**Review by aerorahul — APPROVED** [review]
changes look good.
Thank you for including suggested reviews

**Pull request merged**